tools/releasing/collect_license_files.sh

```diff
@@ -57,4 +57,19 @@ LICENSES="${DST}/licenses"
 [ -f "${LICENSES}" ] && rm -r "${LICENSES}"
 find "${TMP}" -name "licenses" -type d -exec cp -r -- "{}" "${DST}" \;
 
+# Search the source directory and collect those license files that
```

Comment:
The license are in the binary distribution under python/opt, why are we searching the source instead of just the distribution?